var/spack/repos/builtin/packages/r-lars/package.py

```diff
@@ -33,7 +33,7 @@ class RLars(RPackage):
     url      = "https://cran.r-project.org/src/contrib/lars_1.2.tar.gz"
     list_url = "https://cran.rstudio.com/src/contrib/Archive/lars"
 
-    depends_on('r@3.4.0:3.4.9')
+    depends_on('r@2.10:')
```

Comment:
I know you didn't write this package, but can you move this dependency below the versions, and add `type=('build', 'run')`?